engine-rest/engine-rest-openapi/src/main/templates/paths/history/batch/cleanable-batch-report/get.ftl

```diff
@@ -0,0 +1,75 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/batch/get-cleanable-batch-report/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "getCleanableHistoricBatchesReport"
+      tag = "Historic Batch"
+      summary = "Get Cleanable Batch Report"
+      desc = "Retrieves a report about a historic batch operations relevant to history cleanup
+              (see
+              [History cleanup](${docsUrl}/user-guide/process-engine/history/#history-cleanup)
+              ) so that you can tune the history time to live.
+              These reports include the count of the finished batches, cleanable
+              batches and type of the batch.
+              The size of the result set can be retrieved by using the
+              [Get Cleanable Batch Report Count](${docsUrl}/reference/rest/history/batch/get-cleanable-batch-report-count/)
+              method.
+
+              **Please note:**
+              The history time to live for batch operations does not support [Multi-Tenancy](${docsUrl}/user-guide/process-engine/multi-tenancy.md).
+              The report will return an information for all batch operations (for all tenants) if you have permissions
+              to see the history.
+              "
+  />
+
+  "parameters" : [
+
+    <#assign last = false >
+    <#include "/lib/commons/sort-params.ftl">
```

Comment:
Do we need to add that this can be sorted by `finished`?

Reply:
Yes. Here it is required.